src/tools/generate/generate_js.js

```diff
@@ -0,0 +1,12 @@
+const fs = require('fs-extra');
+
+const generate_js = async () => {
+  fs.copy(
+    './node_modules/web-vitals/dist/web-vitals.es5.umd.min.js',
```

Comment:
Actually I've had another look and the [web-vitals repo](https://github.com/GoogleChrome/web-vitals) does not include the `dist` folder by default so this only exists for you as you have manually built it, but won't work for everyone else wihtout adding the package.json dependencies and an extra step.

So our choices are:

1. Build manually periodically
2. Pull in from CDN like unpkg during `npm run generate`. This seems to get the latest version after a redirect: https://unpkg.com/web-vitals/dist/web-vitals.es5.min.js?module
3. Add to package.json and build as part of `npm run generate` before doing this copy.

To be honest I'm not a fan of 3 as we run `npm run generate` on every merge as part of a GitHub Action so adding dependencies, and builds to that just lengthens that. It's not liable to change THAT often to want to take that burden on in every commit.

@max-ostapenko / @rviscomi what are your thoughts on option 2?

Reply:
If I understood the process correctly, it's being built upon `npm install` - I see it locally in /dist folder.
So the copy operation itself takes really no additional effort.

I prefer having it within npm dependencies to pulling from unpkg.com.

Reply:
And a small note, I'm using UMD module for wider compatibility: `web-vitals.es5.umd.min.js`.
Comment if there is something in favour of ESM.

Reply:
> If I understood the process correctly, it's being built upon npm install - I see it locally in /dist folder.

Isn’t that because you ran `npm install` **and** then `browserfy` to do the build? If you delete this folder and then do `npm install` again without `browserfy` does it create the dust folder?

My concern with adding browserfy and the build step, is the length of time it adds to ever commit. Not much I’m sure but it already takes a while to run.

Reply:
After I removed browserify, I cleaned node_modules and reinstalled only required packages from scratch.
So web-vitals installed and fs.copy, nothing else in this branch.

Reply:
Yeah confirmed the same. Weird. Anyway that's a good thing for us! Resolving this.